test/compiler/test.js

```diff
@@ -0,0 +1,74 @@
+var globalizeCompiler = require( "globalize-compiler" );
+var fs = require( "fs" );
+var path = require( "path" );
+var childProcess = require( "child_process" );
+var assert = require( "assert" );
+var glob = require( "glob" );
+
+describe( "compiled", function() {
+	if ( !fs.existsSync( path.join( __dirname, "_compiled" ) ) ) {
+		fs.mkdirSync( path.join( __dirname, "_compiled" ) );
+	}
+	var files = glob.sync( "cases/*.js", {
+		cwd: __dirname
+	} );
+	files.forEach( function(file) {
+		var name = path.basename( file, ".js" );
+		describe( name, function() {
+			it( "should return identical data", function( done ) {
```

Comment:
should return identical output compared to normal globalize? Can we update the description